### ISSUE TITLE
PseudoClasses formatted incorrectly with prefix

### ISSUE THREAD

**Issue by sentilesdal** [muhammadsammy/tailwindcss-classnames#26]
when there is a prefix, the generated class names look like:
```
tw-hover:some-class-name
tw-sm:some-class-name
```

when they ought to look like

```
hover:tw-some-class-name
sm:tw-some-class-name
```

**Comment by tderleth** [comment]
@muhammadsammy to me it looks like this issue is not resolved anymore (or I might do sth. wrong here). 

- Installed version: `2.1.0`
- `tailwind.config.js`: `module.exports = {prefix: 'tw-',...};
- command: `"generate-css-types": "tailwindcss-classnames -i ./tailwind.config.js -o ./src/types/css.ts"`
- produces types like: `tw-hover:tw-bg-opacity-95` instead of `hover:tw-bg-opacity-95`

thanks for help 🙇‍♂️

**Comment by muhammadsammy** [comment]
Can confirm this bug was reintroduced. Thanks, @tderleth!

**Comment by muhammadsammy** [comment]
This was fixed at #213 is released in `v2.1.1`.
Please update it and confirm if there any issues related to this
